.github/workflows/pr.yml

```diff
@@ -339,20 +339,6 @@ jobs:
       - name: Install dependencies (npm ci)
         run: npm ci --prefer-offline
 
-      - name: Install Dot.net
```

Comment:
This isn't necessary for functional tests and was intermittently failing.

Reply:
I added them with the intention of keeping all actions consistently the same...
Else it was getting too difficult to figure out what was reqquired 
hate this copy paste nonense...